cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server_test.go

```diff
@@ -435,7 +436,8 @@ func makeChartRowsJSON(t *testing.T, charts []*models.Chart, pageToken string, p
 	return rowsJSON
 }
 
-func TestGetAvailablePackageSummaries(t *testing.T) {
+// makeServer returns a server backed with an sql mock and a cleanup function
+func makeServer(t *testing.T, authorized bool) (*Server, sqlmock.Sqlmock, func()) {
```

Comment:
Added this helper since two sets of tests were already manually creating the server, and I needed to do so also. One benefit here is that we can now create the server in the body of each test, so the defer cleanup happens for each test, rather than re-using the one mock for all tests.